### PR TITLE
simplify unnecessary groupMap

### PR DESCRIPTION
**Pull request by Congrool**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

No code will use the key of groupMap now, we can simplify it to groupList. Additionally, we can avoid to pubConnectInfo twice to metaGroup when detecting the re-connection between cloud and edge.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: edge/pkg/edgehub/process.go [hunk 1]
```diff
@@ -17,11 +17,10 @@ import (
 	"github.com/kubeedge/kubeedge/edge/pkg/edgehub/config"
 )
 
-var groupMap = map[string]string{
-	"resource": modules.MetaGroup,
-	"twin":     modules.TwinGroup,
-	"func":     modules.MetaGroup,
```
Comment (gy95) [inline edge/pkg/edgehub/process.go:23]: it seems that group `func` is never used, can be deleted directly? 
cc @wackxu

**Comment by Congrool** [comment]
BTW, it seems that EventBus and SeriviceBus do not care about connection state and never handle `messagepkg.ResourceTypeNodeConnectio`. So can we just send it to twin group and meta group?

File: edge/pkg/edgehub/process.go (hunk 1 above)
Comment (wackxu) [reply edge/pkg/edgehub/process.go:23]: yes, we can delete it since we do not have edge function

**Comment by Congrool** [comment]
friendly ping @gy95 , PTAL

**Comment by gy95** [comment]
cc @vincentgoat  PTAL

**Comment by vincentgoat** [comment]
> BTW, it seems that EventBus and SeriviceBus do not care about connection state and never handle msg of type `messagepkg.ResourceTypeNodeConnection`. So can we just send it to twin group and meta group?

agree.
/lgtm

**Pull request closed without merging**